frontend/src/constants.ts

```diff
@@ -1,4 +1,5 @@
-export const REFRESH_TIMER_SECONDS = 30;
+export const REFRESH_TIMER_SECONDS = 33;
```

Comment:
Could you add a comment adding context to why it's set to 33?  Might be useful information for someone in the future :)

Reply:
Yes yes, good call. Done!

Reply:
Good point, Jan. I updated the commented to reflect this.

Reply:
👌